### PR TITLE
Avoid twice serial plotter apps

### PR DESCRIPTION
**Pull request by francescospissu**
### Motivation
IDE2 contains two serial plotter apps, one in the `build` directory and the other in` node_modules/arduino-ide-extension/build`.

### Change description
Do not copy the app into the `build` folder.

### Reviewer checklist

* [ ] PR addresses a single concern.
* [ ] The PR has no duplicates (please search among the [Pull Requests](https://github.com/arduino/arduino-ide/pulls) before creating one)
* [ ] PR title and description are properly filled.
* [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: arduino-ide-extension/src/node/plotter/plotter-backend-contribution.ts [hunk 1]
```diff
@@ -14,7 +14,8 @@ export class PlotterBackendContribution
       '..',
       '..',
       '..',
-      'build',
+      '..',
+      'node_modules',
```
Comment (fstasi) [inline arduino-ide-extension/src/node/plotter/plotter-backend-contribution.ts:18]: I didn't test this, but are you sure this works for the bundled version of the app as well? they have different file locations, I think we cannot include from the node_modules this way...

Comment (francescospissu) [reply arduino-ide-extension/src/node/plotter/plotter-backend-contribution.ts:18]: No, it doesn't work. It just work locally. I'm reverting it.

Comment (kittaakos) [reply arduino-ide-extension/src/node/plotter/plotter-backend-contribution.ts:18]: Let Node.js find the plotter app: [`require.resolve`](https://nodejs.org/api/modules.html#requireresolverequest-options).

See the working example here: https://github.com/arduino/arduino-ide/compare/fspissu/avoid-twice-serial-plotter-app...kittaakos:avoid-twice-serial-plotter-app-2.

Comment (francescospissu) [reply arduino-ide-extension/src/node/plotter/plotter-backend-contribution.ts:18]: I was already investigating to do it in this way. Thanks for the hint @kittaakos 👍

Comment (fstasi) [reply arduino-ide-extension/src/node/plotter/plotter-backend-contribution.ts:18]: nice implementation, using the resolve to leverage how node looks for packages is quite clever!

**Review by fstasi — APPROVED** [review]
lgtm

**Pull request merged**